presto-docs/src/main/sphinx/spill/overview.rst

```diff
@@ -44,6 +44,21 @@ memory intensive queries. It is still possible that the query runner will fail
 to divide intermediate data into chunks small enough that every chunk fits into
 memory, leading to `Out of memory` errors while loading the data from disk.
 
+Disk usage
```

Comment:
What are your thoughts on adding this as a "Recommended hardware configuration" section before the `Memory management and spill` section? An admin is more likely to need these details than how exactly spill works.